atomic_reactor/plugins/build_source_container.py

```diff
@@ -88,7 +91,8 @@ def run(self):
         if maven_source_exists and not os.listdir(maven_source_data_dir):
             self.log.warning("Maven source directory '%s' is empty", maven_source_data_dir)
 
-        image_output_dir = tempfile.mkdtemp()
+        os.makedirs(os.path.join(self.workflow.source.workdir, 'temp'), exist_ok=True)
+        image_output_dir = tempfile.mkdtemp(dir=os.path.join(self.workflow.source.workdir, 'temp'))
```

Comment:
Same here, I would make this `{workdir}/output/`

Reply:
updated.